### PR TITLE
Added: Search-bar in monitoring table

### PR DESCRIPTION
**Pull request by rishav4101**
Description : Search-bar is added in monitoring table for better and improved user experience.
Signed off by : @rishav4101 rajkumarrishav4101@gmail.com
Fixes : #384 
Proof :
Searchbar3
searchbarDarkmode

### CODE REVIEW

**Comment by rishav4101** [comment]
@muskankhedia @Harkishen-Singh Kindly review it.

File: dashboard/v1.1/src/pages/Monitoring/Matrix.tsx [hunk 1]
```diff
@@ -256,6 +272,11 @@ const Matrix: FC<MatrixProps> = ({
   };
   return (
     <>
+      <SearchBar
```
Comment (aquibbaig) [inline dashboard/v1.1/src/pages/Monitoring/Matrix.tsx:275]: Please add proper styling to your components before you request a review. After you have done that, you can re-request a review in the PR settings(if you have access to).

**Review by aquibbaig — CHANGES_REQUESTED** [review]

**Comment by muskankhedia** [comment]
@rishav4101 Can you please add the search bar as implemented over here?

Screenshot 2021-03-21 at 1 35 38 PM

**Comment by muskankhedia** [comment]
@rishav4101 Can you please update the screenshot in the description?

**Comment by muskankhedia** [comment]
@rishav4101 The implementation looks good. Can you please make the search bar compatible with the dark mode?

Screenshot 2021-03-22 at 9 47 10 AM

**Comment by rishav4101** [comment]
@muskankhedia Yes sure will update the code soon. Thank you.

**Comment by rishav4101** [comment]
@muskankhedia I have updated the code, Kindly review it.

**Review by muskankhedia — APPROVED** [review]
LGTM! Thank you for the PR!

**Pull request merged**